kernel/kernel-api/src/main/java/io/delta/kernel/internal/SnapshotImpl.java

```diff
@@ -121,4 +125,33 @@ public Path getLogPath() {
     public Path getDataPath() {
         return dataPath;
     }
+
+    /**
+     * Returns the timestamp of the latest commit of this snapshot.
```

Comment:
Should it say: `Returns the timestamp of when this table snapshot version is created`?

Reply:
It can also be the file modification time as a fallback? So it's not necessarily of the time when this table snapshot version is created?